fonts.css

```diff
@@ -0,0 +1,16 @@
+@font-face {
+  font-family: "GothamPro";
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5542092)